veascan-web/src/components/TxCard.tsx

```diff
@@ -0,0 +1,142 @@
+import React from "react";
+import styled from "styled-components";
+import Copy from "tsx:../../public/tx-info/Copy.svg";
+import ArbitrumLogo from "tsx:../assets/svgs/tx-info/ArbitrumLogo.svg";
+import EthereumLogo from "tsx:../assets/svgs/tx-info/EthereumLogo.svg";
+
+interface TxCardProps {
+  title: string;
+  chain: string;
+  txHash: string;
+  timestamp: string;
+  caller: string;
+  extraFields?: {
+    [key: string]: string;
+  };
+}
+
+const StyledDiv = styled.div`
+  margin: 0px 32px;
+  width: auto;
+  .icon {
+    width: 16px;
+    height: 16px;
+    fill: none;
+    cursor: pointer;
+  }
+
+  .tx-header {
```

Comment:
Abstract all except margin and width to the global-style file, it can be the default for the `label` tag.